### PR TITLE
Website 2.0

### PR DESCRIPTION
**Pull request by abhilashlr**

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+    fastboot: {
```
Comment (gokatz) [inline config/environment.js]: This might affect my development workflow 😄 
I'm using gitpod which will serve app at a custom url.

Comment (abhilashlr) [reply config/environment.js]: Ah! Any suggestions? Or a regex pattern?

Comment (gokatz) [reply config/environment.js]: The regex would be: `/(.*).gitpod.io$/`

However, I'm not sure why fastboot throws an error when I remove the fastboot config itself. Is `hostWhitelist` a mandatory one?

Comment (abhilashlr) [reply config/environment.js]: Looks like, but yes we can add this regex to the list as well?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,11523 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (MalayaliRobz) [inline yarn.lock]: let's have just one lock file, either yarn.lock or package-lock.json

**Pull request merged**